locator/topology.cc

```diff
@@ -6,11 +6,14 @@
  * SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
  */
 
+#include <bit>
+#include <ranges>
```

Comment:
> locator/topology: do_sort_by_proximity: shuffle equal-distance replicas
> To improve balancing when reading in 1 < CL < ALL
> 
> This implementation has a moderate impact on
> the function performance in contrast to full
> std::shuffle of the vector before sorting it
> (especially with large number of nodes to sort).

That doesn't work anyway, since sort() may re-sort them back by address, if the implementation finds it convenient (it's not a stable sort).